test/client/fixtures/sandboxes/event/focus-blur-change.js

```diff
@@ -207,58 +196,56 @@ function testChanging (numberOfHandlers, testCanceled, testCallback) {
     var input2ChangedCount = 0;
 
     var assertChanging = function () {
-        strictEqual(input1ChangeHandlersExecutedAmount, input1ChangedCount, 'input1ChangeHandlersExecutedAmount checked');
-        strictEqual(input2ChangeHandlersExecutedAmount, input2ChangedCount, 'input2ChangeHandlersExecutedAmount checked');
+        return new Promise(function (resolve) {
+            strictEqual(input1ChangeHandlersExecutedAmount, input1ChangedCount, 'input1ChangeHandlersExecutedAmount checked');
```

Comment:
So, asserts are sync actually? Then we don't need to promisify them. We need only async funcs to return promise.